packages/reputation/test/util.js

```diff
@@ -0,0 +1,21 @@
+
+module.exports = {
+    // formats result from PullReputationScore into a dict
+    cleansePullReputationScore: async promise => {
+        let thrown = undefined;
+        try {
+            let result = await promise;
+            return {reviewTotal: result[0],
+            reviewCount: result[1],
+            personalTotal: result[2],
```

Comment:
what's `personal` mean?

Reply:
This is referring to:
`
        signalTypeMapping[0x0] = 'review';   // e.g., host provides review; guest provides a review
        signalTypeMapping[0x1] = 'personal'; // e.g., when a user updates their profile, adds number, etc
        signalTypeMapping[0x2] = 'booking';  // e.g., when a user makes, cancels, rejects a booking
        signalTypeMapping[0x3] = 'misc';     // miscellanous items
`

Reply:
I see, so a user adds to their `reputation` when they add more `personal` information,

Reply:
yup!